src/styles/login.css

```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&family=Montserrat&family=Roboto&display=swap');
+*{
+    padding:0;
+    margin:0;
+}
+html{
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+    
+}
+.landing{
+    width: 95%;
+    margin: 0 auto;
+    display: flex;
+    justify-content:center;
+    align-items: center;
+    flex-direction: row;
+    padding:2rem;
+    
+}
+.landing>.login-wrapper{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+    height: 100%;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)